src/main/java/org/breedinginsight/services/UserService.java

```diff
@@ -297,26 +311,42 @@ public boolean exists(UUID id){
         return dao.existsById(id);
     }
 
-    //TODO: Remove once registration flow is complete
-    public User updateOrcid(AuthenticatedUser activeUser, UUID userId, OrcidRequest orcidRequest) throws DoesNotExistException, AlreadyExistsException {
+    public void createAndSendAccountToken(UUID userId) throws DoesNotExistException {
+
+        BiUserEntity biUser = dao.fetchOneById(userId);
+
+        if (biUser == null) {
+            throw new DoesNotExistException("UUID for user does not exist");
+        }
+
+        SignUpJWT jwt = signUpJwtService.generateNewAccountJWT(userId);
 
-        // This is a temporary fix so any authenticated user is able to update any users orcid.
+        // Save new account token
+        biUser.setAccountToken(jwt.getJwtId().toString());
```

Comment:
Would it be worth (in a different card) storing the date a token/email was generated and sent? I could see this being helpful on the UI to show that an activation token was sent 3 days ago.  And when it is resent, the sent date gets updated.  Thoughts?

Reply:
Yep, I think this is a good idea!

Reply:
I created a card for this:

https://breedinginsight.atlassian.net/browse/BI-665